.github/workflows/testing.yml

```diff
@@ -0,0 +1,33 @@
+name: MindsDB Testing
+
+on:
+  push:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+
+  test-file-handler:
+    name: Test file_handler
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4.7.0
+        with:
+          python-version: 3.9
+      - name: Install python deps
+        run: |
+          pip install .
+          pip install pytest
```

Comment:
@hamishfagg Do we need to install `responses` here too? It looks like the tests fails when importing `responses`

Reply:
we need to install the testing requirements file, which I've added now. We shouldn't be installing any deps individually, because it means out requirements are out of date